src/server.c

```diff
@@ -3730,6 +3730,9 @@ int processCommand(client *c) {
         !obey_client &&
         (is_write_command ||c->cmd->proc == pingCommand))
     {
+        if (!server.repl_ignore_disk_write_error) {
+            serverPanic("Replica was unable to write command to disk.");
+        }
```

Comment:
you must check that this is a replica (LOL).
but also, if it is, and the config is enabled, i'm not sure we should flow to the afterErrorReply flow (to possibly panic there depending onthe other config).
i think we log a warning just once (instead of flooding the log), and make sure not to reach afterErrorReply